### ISSUE TITLE
Add Docs

### ISSUE THREAD

**Issue by odahcam** [smclab/ti-soap#5]
My only hope is go to node.js's soap module and than I get some docs from there.

Maybe a link or warning could be put on README.md to advert about the docs.

_if there's already something, sorry, I didn't saw it._

**Comment by yuchi** [comment]
Yeah we are a little bit elusive on documentation here… also pay attention to the fact that this is based on an old version of ti-soap.

We would be very happy to accept (and publicize!) contributions both to the docs and to updating the module.

Would you want to take some time to make a pull request at least to add some links in the README?

**Comment by yuchi** [comment]
I meant an old version of node-soap

**Comment by odahcam** [comment]
Understood, I will try to make some contribs when I get some time. For sure I will use Soap again and if I have some freetime I will make the contributions.

Thanks.
